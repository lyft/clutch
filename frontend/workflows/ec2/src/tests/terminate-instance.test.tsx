import React from "react";
import { shallow } from "enzyme";

import TerminateInstance from "../terminate-instance";

describe("Terminate Instance workflow", () => {
  let component;

  beforeAll(() => {
    component = shallow(<TerminateInstance resolverType="clutch.aws.ec2.v1.Instance" />);
  });

  it("renders correctly", () => {
    expect(component.debug()).toMatchSnapshot();
  });
});
