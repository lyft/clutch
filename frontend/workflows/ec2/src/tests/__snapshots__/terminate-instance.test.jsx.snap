// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terminate Instance workflow renders correctly 1`] = `
"<Wizard dataLayout={{...}} heading={[undefined]}>
  <InstanceIdentifier name=\\"Lookup\\" resolverType=\\"clutch.aws.ec2.v1.Instance\\" />
  <InstanceDetails name=\\"Modify\\" />
  <Confirm name=\\"Confirmation\\" notes={{...}} />
</Wizard>"
`;
